src/dlgAboutDialog.cpp

```diff
@@ -81,7 +81,7 @@ dlgAboutDialog::dlgAboutDialog(QWidget * parent) : QDialog(parent)
         }
 
         // Repeat for other text, but we know it will fit at given size
-        QString sourceCopyrightText = QChar( 169 ) % QString( " Heiko K" ) % QChar( 246 ) % QString( "hn 2008-" ) % QString(__DATE__).mid(7);
```

Comment:
He hasn't been active for several years and emailed to me that he's not much motivated to do mudlet work - time for us to move on...